src/frontend/screens/Library/components/GameCard/index.tsx

```diff
@@ -121,6 +123,28 @@ const GameCard = ({
           runner
         })
         setGameAvailable(gameAvailable)
+        const nonAvailbleGames = storage.getItem('nonAvailableGames') || '[]'
```

Comment:
hmmm it feels weird to me to store this information in the browser's storage, I feel like the backend should provide that information when listing games instead?

Reply:
I thought about that but at the same time I did not want to change the library if that is what you mean. I think handling the list on frontend it is more intuitive.
What do you propose? use electron-store?

Reply:
hmmm I think what it feels weird to me is that we are storing that information at all, when the source of truth is if the game folder is present or not

maybe a `window.api.listUnavailableGames` or something like that that checks if the game folder for each of the installed games is available and returns the list of games that are not available? we are already asking the backend if a game is available or not

if that's not possible I guess it's fine to store that either in the browser storage or in an electron store

Reply:
I am actually using it as a temporary store just to make the filter work. Because the filter is handled on the frontend. If we want to hide the non-available games on the frontend, that is more dynamic than requesting the list on the frontend. 
Besides, the nature of this call is pretty volatile since it the person can just plug the drive and refresh the library and the state will change.

Also, when starting heroic the store is cleaned up since it will check again which games are available or not.